### PR TITLE
added transactions to accounts page

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/1176

Still need to add the actions for withdraw/deposit/viewtransactions but they require modals that haven't been build yet.

### CODE REVIEW

**Comment by AugurIntegration** [comment]
This pull request **introduces 1 alert** when merging a5d7fe18826c50cf2ad7796d3daff2ae98c70820 into 37a870505ccd2713b199795d219f58706f6d377c - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-f1ede0a0c09798b088182000c17551c6a6c4940c)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/modules/account/components/transactions/transactions-box.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  rep: number | string;
+  gasPrice: string;
+  repFaucet: Function;
+  transferFunds: Function;
+  withdrawReviewModal: Function;
+  closeModal: Function;
+}
+
+export const TransactionsBox = (props: TransactionsBoxProps) => {
+  const showRepFaucet = parseInt(props.augurNodeNetworkId, 10) !== 1;
```
Comment (bthaile) [inline src/modules/account/components/transactions/transactions-box.tsx]: put the is mainnet logic in the container

File: src/modules/account/containers/transactions-box.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import { transferFunds } from "modules/auth/actions/transfer-funds";
+import { selectLoginAccount } from "modules/auth/selectors/login-account";
+import { getGasPrice } from "modules/auth/selectors/get-gas-price";
+import { TransactionsBox } from "modules/account/components/transactions/transactions-box";
+// made state an ANY for now.
+const mapStateToProps = (state: any) => {
+  const loginAccount = selectLoginAccount(state);
+  
+  return {
+    augurNodeNetworkId: state.connection.augurNodeNetworkId,
```
Comment (bthaile) [inline src/modules/account/containers/transactions-box.ts]: just go ahead and convert `state.connection.augurNodeNetworkId,` to IS_MAINNET or allowREPFaucet or something so we don't have `1` in the component.

**Comment by andrewdamelio** [comment]
Good call on the types renames

**Comment by AugurIntegration** [comment]
This pull request **introduces 1 alert** when merging c6a64764bd480250994aed7cb646b524599e32d6 into 6d2eb43c5463efe8f5dcb81e855ef2db0ff5ff76 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-5bcd5e3758645aad6ef37b2e3e8e1d4c53b1572d)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**